features/ubuntu_pro_fips.feature

```diff
@@ -0,0 +1,505 @@
+        log_level: debug
+        log_file: /var/log/ubuntu-advantage.log
+        ua_config:
+          http_proxy: http://<ci-proxy-ip>:3389
+          https_proxy: http://<ci-proxy-ip>:3389
+        """
+        And I verify `/var/log/squid/access.log` is empty on `proxy` machine
+        When I run `ua auto-attach` with sudo
+        Then stdout matches regexp:
+            """
+            SERVICE       ENTITLED  STATUS    DESCRIPTION
+            cc-eal        +yes +<cc-eal-s>  +Common Criteria EAL2 Provisioning Packages
+            """
+        Then stdout matches regexp:
+            """
+            esm-apps      +yes +enabled +UA Apps: Extended Security Maintenance \(ESM\)
+            esm-infra     +yes +enabled +UA Infra: Extended Security Maintenance \(ESM\)
+            fips          +yes +<fips-s> +NIST-certified core packages
+            fips-updates  +yes +<fips-s> +NIST-certified core packages with priority security updates
+            livepatch     +yes +<livepatch-s> +Canonical Livepatch service
+            """
+        Then stdout matches regexp:
+            """
+            <cis_or_usg>           +yes  +<cis-s>  +Security compliance and audit tools
+            """
+        When I run `cat /var/log/squid/access.log` `with sudo` on the `proxy` machine
+        Then stdout matches regexp:
+        """
+        .*CONNECT contracts.canonical.com.*
+        """
+        And stdout does not match regexp:
+        """
+        .*CONNECT metadata.*
+        """
+        Examples: ubuntu release
+           | release | fips-s   | cc-eal-s | cis-s    | livepatch-s | cis_or_usg |
+           | xenial  | n/a      | disabled | disabled | n/a         | cis        |
+           | bionic  | disabled | n/a      | disabled | n/a         | cis        |
+           | focal   | n/a      | n/a      | disabled | enabled     | usg        |
+
+    @series.lts
+    @uses.config.machine_type.aws.pro.fips
+    Scenario Outline: Attached refresh in an Ubuntu pro AWS machine
+        Given a `<release>` machine with ubuntu-advantage-tools installed
+        When I create the file `/etc/ubuntu-advantage/uaclient.conf` with the following:
+        """
+        contract_url: 'https://contracts.canonical.com'
+        data_dir: /var/lib/ubuntu-advantage
+        log_level: debug
+        log_file: /var/log/ubuntu-advantage.log
+        """
+        And I run `ua auto-attach` with sudo
+        And I run `ua status --wait` as non-root
+        And I run `ua status` as non-root
+        Then stdout matches regexp:
+            """
+            .*esm-apps      +yes +enabled +UA Apps: Extended Security Maintenance \(ESM\)
+            esm-infra     +yes +enabled +UA Infra: Extended Security Maintenance \(ESM\)
+            fips          +yes +<fips-s> +NIST-certified core packages
+            fips-updates  +yes +disabled +NIST-certified core packages with priority security updates
+            livepatch     +yes +n/a  +Canonical Livepatch service.*
```

Comment:
Let's also verify that `FIPS` is shown as enabled here by default.
Furthermore, let's also check if FIPS was correctly configured on the machine. Take a look on the FIPS tests we have on either `enable_fips_vm` or `enable_fips_cloud` to see the process we use to check that

Reply:
The machines will be fips enabled by default i.e. the fips machines.